feat: UI improvement

mitosagi opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
Improved the design of the main tab and the plugins & scripts tab. Especially, usability in small window sizes is improved.

## Related Issue
<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
- Resize the window to make sure the design is intact.
- Check the visibility in the dark theme.

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

hal-shu-sato commented:
Overall, it looks good.
But I have 2 points to comment on.

First, when we use a wide window, the content in the AviUtl tab looks too long.
I suggest adding the `.px-md-5 .py-md-4` class into `.card-body`.

Second, I think the tags in the package tab should be separated by a border for legibility.
So, the added emoji is so good, but in terms of the design, the previous one is better, I think.

If you have an intention or reason for such a change, please let us know.

mitosagi commented:
Thank you for your review.

1. reduced the maximum width of the card in the main tab, revert span to input to prevent overflow when the AviUtl folder path is long

2. to increase the visibility of Overview, I tried to make the other elements more moderate. I have reverted because the overall visibility has decreased. Instead of removing the tag border,  I moved it to the right side.

hal-shu-sato reviewed (approved):
Thank you for improving.
That's a better design!
